.github/workflows/deploy-cloud-functions.yml

```
@@ -0,0 +1,75 @@
+          pip install -r cloud_functions/requirements.txt
+          pip install pytest pytest-cov
+
+      - name: Run tests
+        run: |
+          pytest cloud_functions/ --cov=. --cov-report=xml
+
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)